src/com/intellij/plugins/bodhi/pmd/filter/AndFilter.java

```diff
@@ -0,0 +1,27 @@
+package com.intellij.plugins.bodhi.pmd.filter;
+
+import com.intellij.openapi.vfs.*;
+
+class AndFilter implements VirtualFileFilter
+{
+    private final VirtualFileFilter[] filters;
+
+    AndFilter(VirtualFileFilter... filters)
+    {
+        this.filters = filters;
+    }
+
+
+    @Override

```

Comment:
This is also only available on language level 6. In general if you can try to build this with level 5 it will be great.